lib/meta_inspector/parsers/texts.rb

```diff
@@ -45,6 +69,10 @@ def best_description
 
       private
 
+      def find_heading(heading)
+        parsed.css(heading).map { |tag| tag.inner_text.strip.gsub(/\s+/, ' ') }
```

Comment:
I'd add `.reject(&:empty?)` to avoid returning empty strings, what do you think?
```ruby
parsed.css(heading).map { |tag| tag.inner_text.strip.gsub(/\s+/, ' ') }.reject(&:empty?)
```